starboard/doc/eap/getting_started.md

```diff
@@ -0,0 +1,53 @@
+# Open Source Setup
```

Comment:
Can we also link to reference port documentation in perhaps another section?

We have RDK here and can also link to Raspi-2 when available:
- https://github.com/youtube/cobalt/blob/25.lts.1%2B/cobalt/site/docs/development/setup-rdk.md